### PR TITLE
feat: add polynomial trade tvl on polynomial chain

### PR DESCRIPTION
**Pull request by Aniket965**
##### Name (to be shown on DefiLlama):
Polynomial

I want to list polynomial as chain here, do i need to do something else as well i already added polynomial bridge tvl (which is chain tvl)?

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/polynomial-trade exports TVL: 
        
 
 
    optimism                  6.04 M
    polynomial                70.53 k
    
    total                    6.11 M

**Comment by llamatester** [comment]
The adapter at projects/polynomial-bridge exports TVL: 
        
 
 
    ethereum                  1.19 M
    arbitrum                  335.80 k
    optimism                  327.60 k
    base                      179.44 k
    
    total                    2.03 M

**Comment by llamatester** [comment]
The adapter at projects/polynomial-trade exports TVL: 
        
 
 
    optimism                  6.04 M
    polynomial                70.46 k
    
    total                    6.11 M

**Comment by g1nt0ki** [comment]
hi @Aniket965 chain tvl is value of tokens locked in different defi protocols on a given chain, usually much lower than bridge tvl.

File: projects/polynomial-trade/index.js [hunk 1]
```diff
@@ -4,18 +4,31 @@ const { get } = require('../helper/http');
 // api
 const BASE_URL = "https://perps-v2-mainnet.polynomial.fi/snx-perps/tvl";
 
-async function tvl (timestamp, ethBlock) {
+const BASE_URL_POLYNOMIAL_CHAIN = "https://perps-api-mainnet.polynomial.finance/core/portfolio/tvl";
```
Comment (g1nt0ki) [inline projects/polynomial-trade/index.js:7]: what is included as tvl here

Comment (Aniket965) [reply projects/polynomial-trade/index.js:7]: we have smart contract wallet for every user who comes to polynomial trade on polynomial chain they can deposit money in this wallet, and they can deposit as margin in the protocol, we sum up these for all users and return in this api

Comment (Aniket965) [reply projects/polynomial-trade/index.js:7]: makes sense @g1nt0ki ?

Comment (g1nt0ki) [reply projects/polynomial-trade/index.js:7]: sorry, dont think you should count the tvl in smart wallet only the margin deposited in the protocol

Comment (Aniket965) [reply projects/polynomial-trade/index.js:7]: Hey @g1nt0ki i discussed with team, and we removed total balance in there smart contract and only agg the margin we deployed and the api should only show the total of margin of all accounts now

Comment (g1nt0ki) [reply projects/polynomial-trade/index.js:7]: thanks @Aniket965

**Pull request merged**